summerofcode/2023.md

```diff
@@ -73,3 +74,19 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
 - Expected project size: 350h
 - Difficulty: Medium
 - Upstream Issue (URL): https://github.com/knative/eventing/issues/6247
+
+### KubeVela
+
+#### KubeVela HA Enhancement (VelaHA)
+
+- Description: KubeVela can be deployed in a single management cluster.The KubeVela management cluster has a single point of risk. When KubeVela management clusterdisaster occurs, business continuity cannot be guaranteed. In the production environment of enterprise use case(financial industry), KubeVela needs to have higher availability requirements. 
+Consider three enhancements to achieve this goal.
+* Cluster Meta backup & recovery,
+* Master redundancy,
+* Failover support
+- Expected outcome: velaha as addon for KubeVela with command line. cli> vela ha enable 
+- Recommended Skills: Golang, Kubernetes, KubeVela, CUELang
+- Mentor(s): Jiahang Xu @jefree-cat, Xiangbo Ma @fourierr  
```

Comment:
Could you provide email addresses for the mentors on this list? (either here, or you can DM me in slack)

Reply:
ok

Reply:
> Could you provide email addresses for the mentors on this list? (either here, or you can DM me in slack)

ok